### ISSUE TITLE
Children Documentation

### ISSUE THREAD

**Issue by JRJurman** [yelouafi/snabbdom-jsx#16]
Hello, is there any clear documentation on how children are supposed to work in this project?

I'm used to the react-way of rendering children inside components, however, when working with this (in cycle.js), all I get is `undefined`. Some clear documentation would be most appreciated (in the README, or otherwise).

This is the component I'm having trouble rendering, for reference:
```javascript
export const Card = function(attrs, children) {

  return (<div>
    <h2>{attrs.title}</h2>
    {children}
    Check State: {attrs.checkState}
  </div>)
}
```

**Comment by yelouafi** [comment]
can you check with the new version? nested arrays were not handled correctly (which is fixed in 0.3.1 thanks to @ds300)

**Comment by JRJurman** [comment]
Yep, the newest version fixed it!
